pages/blog/release-notes-2.4.0.md

```diff
@@ -0,0 +1,113 @@
+          $ref: "#/components/serverVariables/port"
+  serverVariables:
+    stage:
+      default: dev
+    port:
+      enum: [5000, 6000]
+      default: 5000
+```
+
+These are added to the many other aspects of the AsyncAPI specification which can be declared as reusable components. You can see the full list in the [Components Object section of the AsyncAPI specification](https://www.asyncapi.com/docs/specifications/v2.4.0#componentsObject).
+
+This new feature was contributed by [Daniel Kocot](https://github.com/danielkocot). For more detail, see this [pull request](https://github.com/asyncapi/spec/pull/717) and the [Github issue where this change was discussed](https://github.com/asyncapi/spec/issues/707).
+
+## Security can now be defined at Operation level
+
+Until today, Security requirements were defined at Server level. That restricted the security requirements to be the same for all channels linked with a Server, and for all operations of those channels.
+Thanks to [Sekharbans](https://github.com/sekharbans-ebay), is it possible to increase that granularity by defining a set of security requirements at Operation level. 
+In fact, when setting Security in both the Server and the Operation, both should be satisfied.
+
+For example:
+
+```yaml
+asyncapi: 2.4.0
+servers:
+  production:
+    url: "mykafkacluster.org:8092"
+    protocol: kafka-secure
+    security:
+      - service_auth:
+         - auth:write
+         - auth:read
+channels:
+  some/events:
+    servers:
+      - production
+    subscribe:
+      # This operation level security implies the ability to send a message to 
+      # `some/events` with Authorization headers that have `auth:write` scope. 
+      # Note that the operation level security must still satisfy security options 
+      # specified at the server level.
+      security: 
+       service_auth:
+         - auth:read  
+```
+
+## Reusability of Servers defined in Components is clarified in the specification
+
+Reusability of Servers was introduced in [AsyncAPI 2.3.0](https://www.asyncapi.com/blog/release-notes-2.3.0#servers-and-channels-can-now-be-defined-as-reusable-components). However, the change was not fully clarified in the specification, leading to confusion.
+
+Thanks to [Vladimir Gorej](https://github.com/char0n), this is now clarified by mentioning that elements for the Servers Object can be either Server Object or a Reference Object.
+For more detail, see this [pull request](https://github.com/asyncapi/spec/pull/706) and the [Github issue where this change was discussed](https://github.com/asyncapi/spec/issues/705).
+
+## Tooling support
+
+The following official AsyncAPI tools are already updated to support 2.4.0 version of the specification:
+- JSON Schema that supports validation of AsyncAPI documents is updated in [this](https://github.com/asyncapi/spec-json-schemas) repository. Also **@asyncapi/specs** package has been updated on NPM to version `TBD`, and it contains the 2.4.0 JSON Schema.
```

Comment:
Obviously those versions are TBD

Reply:
Should you add that note to the blog post? I think it is confusing to read your blog post otherwise 🤔

Reply:
Oh yeah, this TBD means I need to change it before publishing the blog post.